features/enums.py

```diff
@@ -18,3 +18,10 @@ class OverrideFieldType(models.TextChoices):
     """Enumeration for overridable fields."""
 
     NAME = "NAME", _("Name")
+
+
+class Visibility(models.IntegerChoices):
+    """Used to show/hide individual features from the API output"""
+
+    VISIBLE = 1, _("Visible")
+    INVISIBLE = 0, _("Invisible")
```

Comment:
Just semantics, but maybe something like "hidden" could be even better instead of invisible.

Reply:
changed to 'hidden'